third_party/tlx/dialect/lib/Transforms/Fixup.cpp

```diff
@@ -26,6 +28,65 @@ class TritonTLXFixupPass : public impl::TritonTLXFixupBase<TritonTLXFixupPass> {
   using impl::TritonTLXFixupBase<TritonTLXFixupPass>::TritonTLXFixupBase;
 
 public:
+  void getBackwardSliceWithWS(Value target,
```

Comment:
Do you think the logic in getBackwardSliceWithWS is generally applicable/useful like mlir::getBackwarsSlice? I was wondering if you should call it something more specific to the logic that you are using to add items to the worklist.